src/app/follower-management-panel/follower-management-panel.component.html

```diff
@@ -12,6 +12,9 @@ <h3>Manage Followers</h3>
       <option [selected]="followerService.sortField === 'Power'">Power</option>
       <option [selected]="followerService.sortField === 'Remaining'">Remaining Life</option>
     </select>
+    <input type="checkbox" [checked]="this.Ascending" (change)="sortAscSwitch()">
```

Comment:
Use followerService.sortAscending here instead of this.Ascending.

Reply:
Another leftover. Hrm. I guess I was on my last legs mentally doing this.